modules/network-observability-sample-network-policy-YAML.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
```

Comment:
@stleerh Previously, I had this YAML as an alternative to filling in the form you can edit the YAML in the `edit YAML` view. After peer review of this documentation, my reviewers pointed out that the YAML was more of a reference than part of a procedure and it was recommended that I move it into a module on its own. So I have it here as a sample network policy. If users want to create a network policy using CLI, they can use this sample as a reference in conjunction with the CLI guidance here: https://docs.openshift.com/container-platform/4.12/networking/network_policy/creating-network-policy.html#nw-networkpolicy-object_creating-network-policy. Does this sound reasonable to you?

Reply:
I think it's fine to move it out of this document.  However, the example seems out of place in the other document as it is listed as the first item on this page.  It should probably be renamed to "Create a network policy ..." where it describes what this policy is trying to do.  Then put it somewhere on that page, but the first item should remain as Creating a networking policy using the CLI, and then all the examples should come after that.

Reply:
Okay, I will move the annotated example after the Creating a network policy for Network Observability procedure.

Reply:
Everything else looks good.  
/lgtm

Reply:
Everything else looks good.  
/lgtm